lib/tsParser.js

```diff
@@ -4,7 +4,14 @@ module.exports = TSParser;
 
 var typedoc = require('typedoc'),
   path = require('path'),
-  TSConstruct = require('./tsConstruct');
+  TSConstruct = require('./tsConstruct'),
+  marked = require('marked');
+
+marked.setOptions({
+  highlight: function(code) {
+    return require('highlight.js').highlightAuto(code).value;
```

Comment:
Hey @ritch I checked how you guys did else where, and try to follow the same, but it didn't work.
Highlight has 2 methods highlight(lang, code), and highlightAuto(code), the only one the that worked was highlightAuto(code), but that too didn't create a good looking markdown. I mean the markedOptions we were using previously with preference set ( gfm: true,
tables: true,
breaks: false,
pedantic: false,
sanitize: false,
smartLists: true,
smartypants: false,
langPrefix: 'lang-',) is not proper to render typescript. A more simpler way of specifying no preference works better.